CHANGELOG.md

```diff
@@ -29,7 +29,7 @@
     upstream services. Thanks [Paul Austin](https://github.com/pauldaustin)
     for the contribution.
     [#2051](https://github.com/Mashape/kong/pull/2051)
-  - logging: Logging plugins now also log the authenticated Consumer.
+  - logging: Logging plugins now also log the `custom_id` or `username` of authenticated Consumers.
```

Comment:
That would not be entirely accurate either. If we wish to rephrase this now, then I would rather say:
```
- Logging plugins: The produced logs now include a `consumer` field, which contains the 
   properties of the authenticated Consumer (`id`, `custom_id`, and `username`), if any.
   [#2367](https://github.com/Mashape/kong/pull/2367)
```

Also: lines in the Changelog should not exceed 80 characters.

Thanks! :)